client/postcss.config.js

```diff
@@ -2,7 +2,10 @@ module.exports = ( { options } ) => {
 	const plugins = { autoprefixer: {} };
 	if ( options.transformCssProperties ) {
 		plugins[ 'postcss-custom-properties' ] = {
-			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
+			preserve: false, // remove custom props from fallback browsers' CSS
+			importFrom: {
+				customProperties: options.customProperties,
```

Comment:
I don't do 
```suggestion
			importFrom: [ require.resolve( '@automattic/calypso-color-schemes/js' ) ],
```

Because `postcss-custom-properties` uses `cosmicconfig`, which in-turn uses `fresh-import`. Fresh import is a `require` function that evaluates the whole module over and over in case config files change during the build (useful for watch mode), a scenario that doesn't apply to us. So I'm importing on a higher level (`webpack.config.js`) and passing the results is faster and does less IO work.